draco/core/containers.py

```diff
@@ -603,3 +603,46 @@ def freq(self):
     @property
     def map(self):
         return self.datasets['map']
+
+class BeamPerturbation(TODContainer):
+    """Container for holding beam perturbation values.
+	(as created in draco.synthesis.expand_perturbed)
+    """
```

Comment:
When I checked this with flake 8 - it had some errors about mixed whitespaces and tab. Did you check those?